### DISCUSSION TITLE
build html5 target is empty folder.

### DISCUSSION THREAD

**Discussion by smallcoderofme** [Kode/Kha#1339] [General]
![image](https://user-images.githubusercontent.com/18281729/118069969-aefba500-b3d7-11eb-8be0-affec607a26c.png)

**Comment by RobDangerous** [comment]
You somehow messed up your Kha directory - when it prints the shrug emoji for the revision that means files are missing.

**Reply by smallcoderofme** [reply]
**Using Kha (e26ec9f5, M Kinc, M Tools/haxe) from F:\workspace_haxe\kha-empty\Kha
Creating Kha project.
Compiling shader 1 of 8 (painter-colored.frag.glsl).
Compiling shader 2 of 8 (painter-image.frag.glsl).
Compiling shader 3 of 8 (painter-image.vert.glsl).
Compiling shader 4 of 8 (painter-text.frag.glsl).
Compiling shader 5 of 8 (painter-text.vert.glsl).
Compiling shader 6 of 8 (painter-video.frag.glsl).
Compiling shader 7 of 8 (painter-video.vert.glsl).
Compiling shader 8 of 8 (painter-colored.vert.glsl).
Type not found : haxe.Exception
Haxe compiler error.**

Is this related to the HAXE version? My version of Haxe is 4.0.5

**Reply by RobDangerous** [reply]
That's not related to your global version of Haxe because that isn't used. It's related to F:\workspace_haxe\kha-empty\Kha\Tools\haxe which your output tells me has been tampered with (see the "M Tools\haxe" in the first line - M stands for modified and there should be no Ms).

**Reply by RobDangerous** [reply]
Hm - also the version of Kha you are using is 1,5 years old.
